src/commands/tree_of.rs

```diff
@@ -88,10 +90,24 @@ pub async fn tree_of(matches: &ArgMatches, repo: Repository, config: &Configurat
                             }
                         )
                     },
-                    &|_, node| format!("label = \"{}\" ", node.1.display_name_version()),
+                    &|_, node| format!("label = \"{}\" ", node.1.clone().display_name_version()),
                 );
 
                 println!("{:?}", dot);
+                Ok(())
+            } else if serial_buildorder {
+                let petgraph: DiGraph<Package, DependencyType> = (*dag.dag()).clone().into();
+
+                let topo_sorted = petgraph::algo::toposort(&petgraph, None)
+                    .map_err(|_| Error::msg("Cyclic dependency found!"))?;
+
+                for i in (0..topo_sorted.len()).rev() {
+                    let node = topo_sorted[i];
+                    let package = petgraph.node_weight(node).unwrap();
```

Comment:
note: We should avoid unwrap if possible but it isn't a big deal in this specific command/part.